ocaml/quicktest/quicktest_common.ml

```diff
@@ -117,7 +98,23 @@ let compare_fields cls fields original_rec new_rec =
     | `Different ->
       if new_field = original_field then
         Alcotest.failf
-          "%s field %s should should be different, but is the same: '%s'"
+          "%s field %s should be different, but is the same: '%s'"
           cls field_name new_field
   in
   List.iter check fields
+
+module Time = struct
+  type t = float
+
+  let now () = Unix.gettimeofday ()
```

Comment:
I think you also have Mtime at your disposal if you want something even more reliable

Reply:
I used stdext-date because that type is stored in the VDI record